.gitignore

```diff
@@ -0,0 +1,5 @@
+# SPDX-FileCopyrightText: 2023 A3 Audio UG (haftungsbeschränkt) <contact@a3-audio.com>
+#
+# SPDX-License-Identifier: CC0-1.0
+
+/home/ra/a3/a3-core/engine/reaper/a3_reaper.RPP-bak
```

Comment:
This path is local to your system and should not be contained in the public repository. Maybe you can create the backup file outside of your working copy instead?